experimental/adaptive-tool/README.md

```diff
@@ -1,57 +1,122 @@
 # Bot Framework Adaptive Tool
-BotFramework adaptive tool is a vscode extension to help developers to improve the fficiency of handering LG/LU files.
+Bot Framework Adaptive Tool is a Visual Studio Code extension that helps developers handle LG (.lg) and LU (.lu) files efficiently.
 
+The support includes:
+- LG/LU syntax highlighting
+- LG/LU diagnostic checks
+- LG/LU completion
+- LG template and function hover
+- LG template definition
+- LG template, function, and structure suggestions
+- Dialog (.dialog) debugging
+- LG debugging
+
+# Quick Start
+## Install from the marketplace
+1. Install the [adaptive-tool](https://marketplace.visualstudio.com/items?itemName=adaptive-tool) from the [Visual Studio Code marketplace](https://marketplace.visualstudio.com/vscode).
+1. Open an LG (.lg) or LU (.lu) file and the adaptive-tool extension will activate.
```

Comment:
>Open an LG (.lg) or LU (.lu) file and the adaptive-tool extension will activate. [](start = 3, length = 80)

Say something about opening .dialog for debugging?

Reply:
the Debugging Feature section contains the steps about how to use the extension to debug the dialog/lg